pulsar-broker/src/main/java/org/apache/pulsar/PulsarStandalone.java

```diff
@@ -246,7 +259,10 @@ public boolean isHelp() {
     @Parameter(names = { "-h", "--help" }, description = "Show this help message")
     private boolean help = false;
 
+    private boolean usingNew211Defaults;
```

Comment:
Using version smells a little from my point ofbview.
What about using PIP117 as reference?
usingPIP117Defaults
or we can invert the flag and name it 'useLegscyDefaults' and cite this PIP with a link